### PR TITLE
Add JENKINS_AGENT_NAME to environment variables

### PR DESCRIPTION
**Pull request by fredrikand**

### CODE REVIEW

**Comment by carlossg** [comment]
is there any reason other than aesthetics ?

**Comment by fredrikand** [comment]
Yes.
The script `jenkins-slave` used in [jenkinsci/docker-jnlp-slave](https://github.com/jenkinsci/docker-jnlp-slave) no longer uses JENKINS_NAME. Since this commit: [96c1c9455df471d7a0c224f8f4ebff98b9e4be9f](https://github.com/jenkinsci/docker-jnlp-slave/commit/96c1c9455df471d7a0c224f8f4ebff98b9e4be9f)

File: README.md [hunk 1]
```diff
@@ -14,7 +14,8 @@ For that some environment variables are automatically injected:
 
 * `JENKINS_URL`: Jenkins web interface url
 * `JENKINS_SECRET`: the secret key for authentication
-* `JENKINS_NAME`: the name of the Jenkins agent
+* `JENKINS_AGENT_NAME`: the name of the Jenkins agent
+* `JENKINS_NAME`: the name of the Jenkins agent (Deprecated. Only here for backwords compatability)
```
Comment (carlossg) [inline README.md]: typo `backwards`

**Comment by carlossg** [comment]
LGTM, just 2 typos

**Pull request merged**